### PR TITLE
Attribution Reporting

### PR DESCRIPTION
**Pull request by murphys0709**

### CODE REVIEW

File: Attribution Reporting.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    priority: <a higher priority number prioritises a source>
+  }),
+);
+```
+See [Attribution-Reporting-Register-Source](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Attribution-Reporting-Register-Source) for a detailed description of all the fields available in this header.
+
+# Attribution Triggers
+Once attribution sources have been registered, attribution triggers must be registered.  An attribution source is eligible for reporting if any page on any of the destination sites triggers attribution for the associated reporting origin.
+
+There are two types of attribution triggers – HTML based and Javascript based.
```
Comment (erik-anderson) [inline Attribution Reporting.md:64]: Nit: blank line after this one.

File: Attribution Reporting.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+## HTML Based Triggers
+HTML-based attribution triggers can be used for detecting conversions on a page when an \<img> or \<script> element loads.  You can register an attribution trigger by adding the attributionsrc attribute to the appropriate element. The registration request is sent to the URL inside the attributionsrc value. If you leave the attributionsrc  value blank, the registration request will be sent to the server the requested resource is hosted on.
```
Comment (erik-anderson) [inline Attribution Reporting.md:66]: Nit: fix double space between "attributionsrc" and "value".

File: Attribution Reporting.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+```
+An XMLHttpRequest with setAttributionReporting() invoked on the request object:
+```
+const attributionReporting = {
+  eventSourceEligible: false,
+  triggerEligible: true,
+};
+
+function triggerMatching() {
+  const req = newXMLHttpRequest();
```
Comment (erik-anderson) [inline Attribution Reporting.md:98]: Space after "new"; "new XMLHttpRequest".

File: Attribution Reporting.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+### Default Value:
+Event source conversion reports are sent by the browser to the ad tech approximately one hour after the event’s expiry.  The default value is 30 days after registration.
+
+Navigation source conversion reports are sent following a more complex schedule of attribution windows. After the navigation, a schedule of built-in reporting windows begins.  By default, navigation reports are sent during one of three windows: 2, 7, or 30 days after registration.  Conversions registered before that deadline are sent at the end of that window.
+
+### Customizable Values:
+There are two values that can be used to customize report transmission times available as fields within the Attribution-Reporting-Register-Source header: report windows and expiration values.
+#### Report Windows:
+Report windows designate  how often reports will be sent.  The fields for customizing these values are:
```
Comment (erik-anderson) [inline Attribution Reporting.md:155]: Nit: Fix double space between "designate" and "how"

File: Attribution Reporting.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+The expiration value designates the time after registration that the attribution source is active. This value defaults to 30 days. The field for customizing this value is:
+
+- expiry: A string representing an expiry time in seconds for the attribution source.
+The soonest value permitted for report windows and expiration values is 1 day. The latest period permitted is 30 days.
+
+### Reporting Delays
+Reports may not be sent exactly at the scheduled dates and times. If the browser isn't running when a report is scheduled to be sent, the report is sent at browser startup. The browser may also decide to delay some of these reports for a random, but short, duration. This duration should be no more than an hour.
+
+## Attribution Reports Data
+To send a report, the browser will make a non-credentialed (i.e. without session cookies) secure HTTP POST request to:
```
Comment (erik-anderson) [inline Attribution Reporting.md:173]: Nit: saying "HTTPS post request" would be more clear than "secure HTTP POST request"

**Review by erik-anderson — CHANGES_REQUESTED** [review]

File: Attribution Reporting.md [hunk 6]
```diff
@@ -0,0 +1,187 @@
+# Attribution Reporting
+## Overview
+Attribution reporting is a strategy that allows marketers and sales teams to track the impact of their marketing efforts on specific goals, such as purchases or sales.  Traditionally, ad conversion measurement relied on third-party cookies, which could be used to track users across sites and raised privacy concerns. To facilitate these measurements without tracking, a new reporting API is required.  Microsoft Edge is supporting a limited version of reporting, with reports being sent per-event. Future work includes replacing these event-level reports with aggregation reporting.
+
+# Attribution Sources
+There are two types of attribution sources - navigation and event sources.  An attribution source is eligible for reporting if any page on any of the associated destination sites (advertiser sites) triggers attribution for the associated reporting origin.
+
+## Navigation Sources (Click)
```
Comment (dmdabbs) [inline Attribution Reporting.md:8]: Are these examples illustrative, or is the "attributionsrc with URL" style the sole supported option?

**Review by erik-anderson — APPROVED** [review]

**Pull request merged**